src/corehost/cli/deps_resolver.h

```diff
@@ -65,9 +60,13 @@ class deps_resolver_t
 
 private:
 
-    bool load();
-
-    bool parse_deps_file(const arguments_t& args);
+    static pal::string_t get_fx_deps(const pal::string_t& fx_dir, const pal::string_t& fx_name)
+    {
+        pal::string_t fx_deps = fx_dir;
+        pal::string_t fx_deps_name = pal::to_lower(fx_name) + _X(".deps.json");
```

Comment:
I think Ubuntu is case-sensitive, so if we lower the FX_name and use it later to lookup the file, it may fail. Can you please confirm this is not an issue?